### PR TITLE
added shimmer_evm core assets and wrapped shimmer tokenMapping

### PR DESCRIPTION
**Pull request by simone1999**

### CODE REVIEW

File: projects/helper/coreAssets.json [hunk 1]
```diff
@@ -1414,5 +1414,10 @@
     "WETH": "0x4200000000000000000000000000000000000001",
     "USDC": "0x49A5010110a358d9069282873F3e7eCf6B41DC10",
     "USDT": "0x0Cf7c2A584988871b654Bd79f96899e4cd6C41C0"
+  },
+  "shimmer_evm": {
+    "WSMR": "0xBEb654A116aeEf764988DF0C6B4bf67CC869D01b",
+    "USDT": "0xc0E49f8C615d3d4c245970F6Dc528E4A47d69a44",
```
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:1420]: hi @simone1999 how is the USDT bridged here, also, can you remove ICE from this list, I try to keep it minimal, I will probably try to add another file called address book for tokens not used for pricing other tokens

File: projects/helper/tokenMapping.js
Comment (g1nt0ki) [inline projects/helper/tokenMapping.js:1]: this change is good

File: projects/helper/coreAssets.json (hunk 1 above)
Comment (simone1999) [reply projects/helper/coreAssets.json:1420]: Hey, will remove the ICE. The USDT is bridged via the IceCreamSwap.com bridge, it is the BSC usdt and the bridge is live on 9 chains and running for over 1 year with over $1 million of locked tokens. I think it currently is the only stable coin bridge for shimmer Evm.

**Comment by g1nt0ki** [comment]
included your changes here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7614/files

**Pull request closed without merging**